include/glow/Importer/ModelLoader.h

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef GLOW_IMPORTER_MODELLOADER_H
+#define GLOW_IMPORTER_MODELLOADER_H
+
+#include "glow/Base/Tensor.h"
+#include "glow/Graph/Graph.h"
+
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/Support/raw_ostream.h"
+
+#include <google/protobuf/text_format.h>
```

Comment:
Can we not include this file in the header file?

Reply:
I tried to do this, but couldn't. `unexpectedNodeError()` implementation requires this. Apparently implementation can't live in .cpp file because it's template function. (I followed [this](https://stackoverflow.com/questions/3040480/c-template-function-compiles-in-header-but-not-implementation) as a confirmation that template function can't be in cpp, please let me know if there is a way)